### DISCUSSION TITLE
How do I use this `cycle-profile.lua` to restore a profile?

### DISCUSSION THREAD

**Discussion by WhoMI7** [CogentRedTester/mpv-scripts#19] [Q&A] [answered]
I wanna use something like `script-message cycle-profiles  ";gpu-hq;gpu-hq restore;"`. But gives me `Unknown profile 'gpu-hq restore'.` I can use `apply-profile gpu-hq restore` as a bind to set them but the problem occurs when I have to use `" "`.

**Comment by CogentRedTester** [comment, marked as answer]
I'm not really maintaining `cycle-profile` any more. I'd suggest using [cycle-commands.lua](https://github.com/CogentRedTester/mpv-scripts/blob/master/cycle-commands.lua) instead. You could use:

```properties
Ctrl+t script-message cycle-commands 'apply-profile gpu-hq' 'apply-profile gpu-hq restore'
```

Or if you wanted to maintain the osd message:

```properties
Ctrl+t script-message cycle-commands "apply-profile gpu-hq ; show-text 'applying profile gpu-hq'" "apply-profile gpu-hq restore ; show-text 'restoring profile gpu-hq'"
```

**Reply by WhoMI7** [reply]
This is awesome. Thanks.
